tests/src/checkPythonConversion.cpp

```diff
@@ -0,0 +1,26 @@
+//
```

Comment:
rename file to pythonConversionTest to make it consistent with others?

Reply:
yes thanks